include/fuzzuf/algorithms/afl/templates/afl_state.hpp

```diff
@@ -46,7 +46,7 @@ template <class Testcase>
 AFLStateTemplate<Testcase>::AFLStateTemplate(
     std::shared_ptr<const AFLSetting> setting,
     std::shared_ptr<executor::AFLExecutorInterface> executor,
-    std::unique_ptr<optimizer::Optimizer<u32>>&& _mutop_optimizer)
+    std::shared_ptr<optimizer::Optimizer<u32>>&& _mutop_optimizer)
```

Comment:
I think there is no longer the reason to use rvalue reference now that we replaced unique_ptr with shared_ptr.

Reply:
fixed in 6b50cebeecd293f89664bc2a8165c3b89bc7ff35